### DISCUSSION TITLE
mkdocs serve --dirtyreload does not work - corresponding page is not updated

### DISCUSSION THREAD

**Discussion by MaximBashurov** [mkdocs/mkdocs#3248] [Q&A]
After
`mkdocs serve --dirtyreload`
Update and save md - corresponding page does not update in a browser. How can this be fixed?

**Comment by pawamoy** [comment]
Is your page a symlink to another file?

**Reply by MaximBashurov** [reply]
No, it is not. And this situation are equal for any site pages I edit and save - no updates of these pages take place.

**Reply by pawamoy** [reply]
OK then we'll need more information (OS? Docker?) and most probably a reproducible, minimal example.

**Reply by MaximBashurov** [reply]
No Docker. OS = macOS Ventura 13.1
Client = Visual Studio Code 1.79.2 (Universal), built in terminal = zsh.

INFO     -  [20:02:55] Detected file changes
INFO     -  Building documentation...
INFO     -  Documentation built in 0.56 seconds
INFO     -  [20:02:56] Reloading browsers
INFO     -  [20:02:56] Browser connected: page address

Page is opened in Google Chrome. Does not update. Old version.

**Reply by pawamoy** [reply]
We will need the version of MkDocs you are using, as well as your configuration file (`mkdocs.yml`), and the `docs` folder containing your Markdown pages (that's what I meant by "reproducible, minimal example").
